### PR TITLE
[bench] Add code to run multiple command lines and export the result in a csv file

### PR DESCRIPTION
**Pull request by xadupre**

### CODE REVIEW

File: onnxscript/tools/benchmark/benchmark_run.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+import sys
+
+
+class BenchmarkError(RuntimeError):
+    pass
+
+
+def get_machine() -> dict[str, str | int | float | tuple[int, int]]:
+    """Returns the machine specification."""
+    cpu: dict[str, str | int | float | tuple[int, int]] = dict(
```
[the remark this answers is not in the document]
Comment (gramalingam) [reply onnxscript/tools/benchmark/benchmark_run.py:20]: or `machine_config` or `config` etc.

**Review by gramalingam — APPROVED** [review]

File: onnxscript/tools/benchmark/benchmark_run.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (xadupre) [reply onnxscript/tools/benchmark/benchmark_run.py:20]: renamed into config

File: onnxscript/tools/benchmark/benchmark_helpers.py [hunk 2]
```diff
@@ -195,6 +196,51 @@ def run_benchmark(
...
+) -> tuple[float, float]:
+    """
+    Computes the discrepancies.
+
+    Args:
+        expected: list of outputs coming from a torch model
+        outputs: list of outputs coming from an onnx model
+
+    Returns:
+        max absole errors, max relative errors
```
Comment (gramalingam) [inline onnxscript/tools/benchmark/benchmark_helpers.py:210]: nit: => "absolute"

**Review by gramalingam — APPROVED** [review]

File: onnxscript/tools/benchmark/benchmark_helpers.py (hunk 2 above)
Comment (xadupre) [reply onnxscript/tools/benchmark/benchmark_helpers.py:210]: done

File: onnxscript/tools/benchmark/benchmark_helpers.py [hunk 3]
```diff
@@ -195,6 +196,52 @@ def run_benchmark(
     return data
 
 
+def measure_discrepancies(
+    expected: list[tuple(Any, ...)],
```
Comment (justinchuby) [inline onnxscript/tools/benchmark/benchmark_helpers.py:200]:
```suggestion
    expected: list[tuple[Any, ...]],
```

Comment (xadupre) [reply onnxscript/tools/benchmark/benchmark_helpers.py]: done

**Pull request merged**